<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGLBaseApp</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="GLWidget" name="glWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuControl_Panels">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFrontView"/>
    <addaction name="actionSideView"/>
    <addaction name="actionTopView"/>
    <addaction name="actionToggleOrthographic"/>
    <addaction name="separator"/>
    <addaction name="actionToggleRenderSettings"/>
    <addaction name="actionToggleCameraControls"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControl_Panels"/>
  </widget>
  <widget class="QDockWidget" name="dockFrameProperties">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="windowTitle">
    <string>Object Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="loadModelButton">
         <property name="text">
          <string>Load Model</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="saveModelButton">
         <property name="text">
          <string>Save Model</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QtTreePropertyBrowser" name="propertiesBrowser" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="assignMarkersButton">
         <property name="text">
          <string>Assign Markers</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="fitModelButton">
         <property name="text">
          <string>Fit Model</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSlider">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QSlider" name="captureFrameSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockModelStateEditor">
   <property name="windowTitle">
    <string>Model State</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="loadModelStateButton">
         <property name="text">
          <string>Load State</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveModelStateButton">
         <property name="text">
          <string>Save State</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QtTreePropertyBrowser" name="modelStateEditor" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionToggleRenderSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Render Settings</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionTogglePlayerControls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Player Controls</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionReloadFiles">
   <property name="text">
    <string>Reload Files</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFrontView">
   <property name="text">
    <string>Front View</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionSideView">
   <property name="text">
    <string>Side View</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionTopView">
   <property name="text">
    <string>Top View</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionOrthographic">
   <property name="text">
    <string>Orthographic</string>
   </property>
  </action>
  <action name="actionPerspective">
   <property name="text">
    <string>Perspective</string>
   </property>
  </action>
  <action name="actionToggleOrthographic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Orthographic</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionToggleCameraControls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera Controls</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <zorder>dockFrameProperties</zorder>
  <zorder>dockWidgetSlider</zorder>
  <zorder>dockModelStateEditor</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">GLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">QtTreePropertyBrowser</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
